### PR TITLE
Port daily-validation scripts for bridge plus adjust for new build style and model test script changes

### PR DESCRIPTION
**Pull request by crlishka**
The retiring of ngraph-tensorflow-1.3 affected the ngraph-tensorflow daily-validation job.  This PR is for porting the daily-validation from ngraph-tensorflow-1.3 to ngraph-tensorflow-bridge.

This is a sibling PR to a cje-algo PR that will be opened soon.  (I will update this description at that time.)

A few changes require that adjustments are needed to the repo-side daily-validation testing:

- Changes to the model python scripts (mlp-mnist, resnet20) required adjustments to the pytests that run them

- We are now running resnet20-cifar10 (instead of resnet8), and comparing _accuracy_.  The resnet20-cifar10 pytest has been adjusted to report this, as well as save both accuracy and loss in the JSON run stats (see below).

- The ngraph build is now handled differently, so the job build system needs to be adapted

- The directory structure in the bridge repo (vs ng-tf-1.3) is completely changed, and Christian's maint scripts were replaced by a simpler system

- Several silent-failures were found in installing the plugins directory.  As a result, more sanity checks (including running install_test.py) have been added.

Note besides modifying the validation model-pytests and the Docker testing system, I have also made a minor change to **install_test.py**.  My change adds meaningful exit codes when install_test.py is run as a script, and does not affect when it is used as a library.

I have also added a system to report and save out run stats in JSON files.  This is a precursor to being able to read in reference results (in JSON form) from prior reference runs.  While this PR does not add the capability to read JSON reference results yet, I do intend to add that capability (under a separate Jira task) after this PR is merged.

### CODE REVIEW

File: test/ci/lib_validation_testing.py [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+
+    for f in dataFiles:
+        if not os.path.isfile(os.path.join(dataDir, f)):
+            raise Exception('Data file %s not found in %s'
+                            % (f, dataDir))
+
+# End: checkMnistData()                          
+
+
+def checkNGraphEnvironment():
```
Comment (avijit-nervana) [inline test/ci/lib_validation_testing.py]: We shouldn't need the LD_LIBRARY_PATH any more. If so then please remove this.

File: test/install_test.py [hunk 2]
```diff
@@ -48,6 +48,8 @@ def check_for_ngraph_device():
     [found, name] = check_for_ngraph_device()
     if not found:
         print("nGraph Device is not available")
+        sys.exit(1)
```
Comment (avijit-nervana) [inline test/install_test.py]: Looks good.

File: test/ci/lib_validation_testing.py (hunk 1 above)
Comment (crlishka) [reply test/ci/lib_validation_testing.py]: I found when installing the tensorflow-wheel, then installing the plugins into the installed tensorflow, that LD_LIBRARY_PATH did have to be set.  Let's root-cause this issue outside this PR, and if necessary open a separate Jira issue to handle any changes that need to be made.

Comment (avijit-nervana) [reply test/ci/lib_validation_testing.py]: Sounds good. Let's get this PR merged.

**Pull request merged**